packages/material-ui-styles/src/createGenerateClassName.js

```diff
@@ -60,7 +60,7 @@ export default function createGenerateClassName(options = {}) {
     }
 
     // Help with debuggability.
-    if (styleSheet && styleSheet.options.classNamePrefix) {
+    if (styleSheet.options.classNamePrefix) {
```

Comment:
Is there a possibility that it might be intended that no `stylesheet` is present?

Reply:
We have some code before that assume the `styleSheet` is always present. This change is embracing it fully.